flink-table/flink-table-planner-blink/src/test/resources/org/apache/flink/table/planner/plan/batch/sql/UnnestTest.xml

```diff
@@ -34,7 +34,7 @@ LogicalProject(a=[$0], s=[$3])
     <Resource name="planAfter">
       <![CDATA[
 Calc(select=[a, f0 AS s])
-+- Correlate(invocation=[explode($cor0.c)], correlate=[table(explode($cor0.c))], select=[a,b,c,f0], rowType=[RecordType(INTEGER a, BIGINT b, VARCHAR(2147483647) ARRAY c, VARCHAR(2147483647) f0)], joinType=[INNER])
```

Comment:
Is `explode` the legacy name and that's why we're renaming them now? That's probably independent of refactoring `UNNEST` to use the new type system, isn't it?

Reply:
I don't know why we called it `explode` in the past. There is no reason for it. The official term for this function in SQL is `UNNEST` and it is rather confusing to not show this in the plan as well.